modules/images-configuration-registry-mirror-configuring.adoc

```diff
@@ -30,15 +42,26 @@ For example, after installing the skopeo RPM package on a Red Hat Enterprise Lin
 +
 [source,terminal]
 ----
-$ skopeo copy \
+$ skopeo copy --all \
 docker://registry.access.redhat.com/ubi9/ubi-minimal:latest@sha256:5cf... \
 docker://example.io/example/ubi-minimal
 ----
 +
 In this example, you have a container image registry that is named `example.io` with an image repository named `example` to which you want to copy the `ubi9/ubi-minimal` image from `registry.access.redhat.com`. After you create the mirrored registry, you can configure your {product-title} cluster to redirect requests made of the source repository to the mirrored repository.
 
-. Log in to your {product-title} cluster.
+ifdef::winc[]
+* You must mirror the `mcr.microsoft.com/oss/kubernetes/pause:3.9` image. For example, you could use the following `skopeo` command to mirror the image:
```

Comment:
I was wondering why this is a bullet since the step 1 is "to do this, you must either..." and then there are already 2 bullets. Maybe this would just be an indent, no bullet?